java-spiffe-core/src/test/java/io/spiffe/svid/jwtsvid/JwtSvidParseAndValidateTest.java

```diff
@@ -119,119 +131,159 @@ void testParseAndValidate_nullAudience_throwsNullPointerException() throws JwtSv
 
         return Stream.of(
                 Arguments.of(TestCase.builder()
-                        .name("1. success using EC signature")
+                        .name("1. using EC signature")
```

Comment:
Consider just removing the numbers from all these test names since they inevitably change whenever test cases are added/removed. It results in a lot of noise in diffs and manual work to fix the test name numbers. I don't really see too much value in building in ordering into the names.

This can also be done as a separate PR since I see this convention is already adopted in many of the tests.

Reply:
I think you are right, it helped when debugging failing tests, but it produces noise and manual work when adding new tests. I'll remove them in these two test classes I touched in this PR and will address the rest in a separate PR.